### ISSUE TITLE
Broken file path handling on Windows

### ISSUE THREAD

**Issue by axxelG** [goreleaser/nfpm#260]
It seems like you cannot add files to packages on windows anymore.

I'm almost certain it worked some years ago with this config I used in an old project.
```
files:
  "./install/systemd_loxwebhook.service": "/etc/systemd/system/loxwebhook.service"
```

I tested different config values today but none worked:
```
files:
  "install\\systemd_loxone-exporter.service": "/etc/systemd/system/loxone-exporter.service"

nfpm.exe: error: glob failed: install\systemd_loxone-exporter.service: no matching files

--------------------------------------------------------

files:
  "./install/systemd_loxone-exporter.service": "/etc/systemd/system/loxone-exporter.service"

nfpm.exe: error: glob failed: ./install/systemd_loxone-exporter.service: no matching files

--------------------------------------------------------

files:
  ".\\install\\systemd_loxone-exporter.service": "/etc/systemd/system/loxone-exporter.service"

nfpm.exe: error: glob failed: .\install\systemd_loxone-exporter.service: no matching files

--------------------------------------------------------

files:
  "C:\\Users\\axel.gluth\\go\\src\\github.com\\axxelG\\loxone-prometheus-exporter\\install\\systemd_loxone-exporter.service": "/etc/systemd/system/loxone-exporter.service"     

nfpm.exe: error: matching "C:Users\\axel.gluth\\go\\src\\github.com\\axxelG\\loxone-prometheus-exporter\\install\\systemd_loxone-exporter.service": file does not exist
```

I tried to take a look at the code but the globbing part is quite complex and even if I would find a solution I would have no idea if this breaks on other operating systems or at other places.
Side note: The tests for fileglob have hardcoded `/` while the code uses `filepath.Separator`

If the way to fix this is clear I will most probably be able to provide a PR or at least test changes on windows.

**Comment by axxelG** [comment]
With `disable_globbing: true`
It seems to works with `install\\systemd_loxone-exporter.service": "/etc/systemd/system/loxone-exporter.service`

I failed to pass this setting from goreleaser but I just took a quick look. I will try to take a deeper look later.

**Comment by axxelG** [comment]
I just wanted to take a look but it seems like you already fixed it with #262 
It does not work in goreleaser but @caarlos0 wrote "LGTM so far, once this is fixed/released, will do the same on goreleaser." so I won't create a separate issue in goreleaser for that. I will just wait until it's implemented there.

Thank you for fixing this.

I like to keep this issue open until the fix is in a released version. Feel free to close it if you prefer that.

**Comment by caarlos0** [comment]
I _think_ this is now fixed... will close.
